spec/requests/async_info_spec.rb

```diff
@@ -9,27 +9,19 @@
 
   describe "GET /async_info/base_data" do
     context "when not logged-in" do
-      before { get "/async_info/base_data" }
-
-      it "returns broadcast" do
-        expect(response.body).to include("broadcast")
-      end
-
-      it "returns token" do
-        expect(response.body).to include("token")
-      end
-
-      it "does not return user" do
-        expect(response.body).not_to include("user")
+      it "returns json without user" do
+        get "/async_info/base_data"
+        expect(response.parsed_body.keys).to match_array(%w[broadcast param token])
```

Comment:
this is unrelated spec optimization for a different caching code. Hope that's okay to sneak in